torch/nn/modules/container.py

```diff
@@ -87,8 +87,11 @@ def __dir__(self):
         keys = [key for key in keys if not key.isdigit()]
         return keys
 
+    def __iter__(self):
+        return iter(self._modules.values())
+
     def forward(self, input):
-        for module in self._modules.values():
+        for module in self:
```

Comment:
From benchmarking it was unclear if changing from `for module in self._modules.values()` to `for module in self` had any affect...